### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by theasteve**

### CODE REVIEW

**Comment by theasteve** [comment]
Screen Shot 2021-02-18 at 11 50 44 AM

File: src/twig/partials/footer.twig [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
+          <span class="sr-only">The Mayor's Office for Economic Opportunity</span>
+        </a>
+      </div>
+      <div class="pb-4 desktop:pl-4 desktop:flex justify-end items-center">
+        <nav class="o-footer__inline-nav">
+          <a href="index">Home</a>
+          <a href="https://github.com/CityOfNewYork/nyco-patterns">GitHub Repository</a>
+          <a href="https://github.com/CityOfNewYork/nyco-patterns-framework">NYCO Patterns Framework</a>
+          <a href="https://github.com/topics/nycopportunity?q=org%3Acityofnewyork+topic%3Anycopportunity&amp;unscoped_q=org%3Acityofnewyork+topic%3Anycopportunity">NYCO Opportunity on GitHub</a>
+        </nav>
```
Comment (devowhippit) [inline src/twig/partials/footer.twig:18]: @theasteve These links are only relevant to NYCO patterns. We need to swap these footer links with the links from the original footer.

**Review by devowhippit — CHANGES_REQUESTED** [review]
Looks good; once the links are restored it should be good to go.

File: src/twig/partials/footer.twig (hunk 1 above)
Comment (theasteve) [reply src/twig/partials/footer.twig:18]: Ah didn't catch that, will make changes now https://media.giphy.com/media/3o7ZesJdQccI0sO3fy/giphy.mp4

**Comment by theasteve** [comment]
Screen Shot 2021-02-18 at 3 09 00 PM

Footer links have been updated.

**Review by devowhippit — APPROVED** [review]

**Pull request merged**